libafl/src/mutators/mutations.rs

```diff
@@ -475,9 +489,12 @@ where
             return Ok(MutationResult::Skipped);
         }
 
-        let off = state.rand_mut().below(size as u64) as usize;
-        let len = state.rand_mut().below((size - off) as u64) as usize;
-        input.bytes_mut().drain(off..off + len);
+        let range = rand_range(state, size, size);
```

Comment:
@VTCAKAVSMoACE is `rand_range(state, size, size);` correct?

Reply:
Yes, it is. The range can be up to the full size of the input.

Perhaps we should look into selecting a size bracket first (e.g. small, medium, large, all), then making a range of that size bracket?